### DISCUSSION TITLE
Is this the most recent files

### DISCUSSION THREAD

**Discussion by SW1F7Y-101** [pardeike/Harmony#518] [Q&A]
I'm trying to use Harmony to install the Color coded mood bar on Rimworld, I'm kinda new to pc gaming, I have a very limited knowledge of mods and where to find them and all that lol. please help me XD

**Comment by pardeike** [comment]
This is the wrong repository. Use this instead: https://github.com/pardeike/HarmonyRimWorld

**Reply by SW1F7Y-101** [reply]
Ah thank you I did look at that, but it confused me and is still confusing me, the bit about the Harmony Library mod. like I dont understand what im meant to do there, and does it apply to the mod i want to use? i assume its part of it?

and then the part where I add the text into the xml file, i dont know where to add it lol

**Comment by pardeike** [comment]
Please use the official Rimworld discord for questions like that.
